content/en/infrastructure/containers/_index.md

```diff
@@ -73,38 +75,40 @@ spec:
 {{% /tab %}}
 {{% tab "Helm" %}}
 
-If you are using the [official Helm chart][1], enable the `processAgent.enabled` parameter in your [`values.yaml`][2] file:
+If you are using the [official Helm chart][1], container collection is enabled by default.
+
+For verification, ensure that the `processAgent.containerCollection` parameter is set to `true` in your [`values.yaml`][2] file:
 
 ```yaml
 datadog:
   # (...)
   processAgent:
-    enabled: true
+    containerCollection: true
 ```
 
 Then, upgrade your Helm chart.
 
-In some setups, the Process Agent and Cluster Agent cannot automatically detect a Kubernetes cluster name. If this happens, the feature does not start, and the following warning displays in the Cluster Agent log: `Orchestrator explorer enabled but no cluster name set: disabling.` In this case, you must set `datadog.clusterName` to your cluster name in `values.yaml`.
+In some setups, the Cluster Agent cannot automatically detect a Kubernetes cluster name. If this happens, the feature does not start, and the following warning displays in the Cluster Agent log: `Orchestrator explorer enabled but no cluster name set: disabling.` In this case, you must set `datadog.clusterName` to your cluster name in `values.yaml`.
```

Comment:
Remove mention of dedicated process agent as container collection functionality no longer runs in process agent by default